mappings/net/minecraft/server/function/CommandFunctionManager.mapping

```diff
@@ -1,40 +1,83 @@
 CLASS net/minecraft/class_2991 net/minecraft/server/function/CommandFunctionManager
-	FIELD field_13412 LOAD_FUNCTION Lnet/minecraft/class_2960;
-	FIELD field_13417 TICK_FUNCTION Lnet/minecraft/class_2960;
+	COMMENT The command function manager implements execution of functions, like that from
+	COMMENT the {@code function} command.
+	FIELD field_13412 LOAD_TAG_ID Lnet/minecraft/class_2960;
+	FIELD field_13417 TICK_TAG_ID Lnet/minecraft/class_2960;
 	FIELD field_13418 tickFunctions Ljava/util/List;
+		COMMENT A list of {@code minecraft:tick} tag functions to run on every tick. Set up on
+		COMMENT load, this is more efficient than polling the tag from the {@link #loader}
+		COMMENT every tick.
 	FIELD field_13419 server Lnet/minecraft/server/MinecraftServer;
-	FIELD field_13422 needToRunLoadFunctions Z
+	FIELD field_13422 justLoaded Z
+		COMMENT Whether this command function manager has just {@linkplain #load(FunctionLoader)
+		COMMENT loaded} and should run all functions in the {@code minecraft:load} function tag.
 	FIELD field_25333 loader Lnet/minecraft/class_5349;
-	FIELD field_33542 NO_RECURSION_TEXT Lnet/minecraft/class_2561;
+		COMMENT The source of functions for this command function manager.
+	FIELD field_33542 NO_TRACE_IN_FUNCTION_TEXT Lnet/minecraft/class_2561;
+		COMMENT A localized piece of text indicating that calling the debug command to debug
+		COMMENT functions from within a function is not supported.
+	FIELD field_33543 execution Lnet/minecraft/class_2991$class_6345;
+		COMMENT The active execution within this manager.
 	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/class_5349;)V
 		ARG 1 server
 		ARG 2 loader
-	METHOD method_12899 getTaggedFunctionSource ()Lnet/minecraft/class_2168;
+	METHOD method_12899 getCommandSource ()Lnet/minecraft/class_2168;
+		COMMENT {@return the command source to execute functions}
+		COMMENT
+		COMMENT <p>This command source {@linkplain ServerCommandSource#hasPermissionLevel(int)
```

Comment:
Is this affected by function-permission-level in server.properties?

Reply:
That is only used when parsing functions. I need to look into brigadier more for how exactly it works.